### DISCUSSION TITLE
Please specify required key vault certificate permissions in docs

### DISCUSSION THREAD

**Discussion by abydal** [polymind-inc/acmebot#357] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/5933904/132958872-83ab7b42-7c33-46e5-ad90-28f8cfde57e0.png)

Maybe all 15 permissions are needed, but if not it would be great to know which ones that can be omitted.

**Comment by shibayan** [comment, marked as answer]
I do not plan to add detailed permissions in the documentation, as I can foresee my support costs increasing. And I plan to recommend RBAC in the future.

I have not tried it, but it will probably work with only the permissions. `get` / `list` / `create` / `update`
